alpine-consul/root/etc/services.d/consul/run

```diff
@@ -13,5 +13,15 @@ consul-available
 # We have another Consul server available. Join it.
 RETRY_JOIN=`consul-join`
 
+EXTRA=""
```

Comment:
Could this stuff be moved into a script similar to `consul-join`? We could call it `consul-join-wan` and it would return nothing by default, otherwise it would return everything as below. I think this would make it cleaner and allow one to overwrite that file to customise WAN specific config easily as required.